Formula/brpc.rb

```diff
@@ -0,0 +1,63 @@
+class Brpc < Formula
+  desc "Better RPC framework"
+  homepage "https://brpc.apache.org/"
+  url "https://dlcdn.apache.org/incubator/brpc/1.3.0/apache-brpc-1.3.0-incubating-src.tar.gz"
+  sha256 "582287922f5c8fe7649f820a39f64e1c61c3fdda827c7b393ad3ec2df5b4f9d7"
+  license "Apache-2.0"
+  head "https://github.com/apache/incubator-brpc.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "gflags"
+  depends_on "leveldb"
+  depends_on "openssl@3"
+  depends_on "protobuf"
+
+  def install
+    args = %w[
+      -DBUILD_SHARED_LIBS=ON
+      -DBUILD_UNIT_TESTS=OFF
```

Comment:
why not also turn off `WITH_DEBUG_SYMBOLS`?

Reply:
- `WITH_DEBUG_SYMBOLS`: Sorry, I must have missed that.
- `DOWNLOAD_GTEST`: Because that option will only be considered when `BUILD_UNIT_TESTS` is switched on. But to make it clear, I'll turn that off too.

Reply:
👍